### PR TITLE
[DEX-14398] - Creation of a new LP for FR displaying DIP and PSP

### PR DESCRIPTION
**Pull request by Tiseanu**
Please always provide the [GitHub issue(s)](../issues) your PR is for, as well as test URLs where your change can be observed (before and after):

Fix #<gh-issue-id>

Test URLs:
- Before: https://main--helix-poc--enake.hlx.page/consumer/fr/new/dip-premiumsecurity-opt
- After: https://<branch>--helix-poc--enake.hlx.page/consumer/fr/new/dip-premiumsecurity-opt

### CODE REVIEW

File: blocks/b-big-carousel-quotes/b-big-carousel-quotes.css
Comment (enake) [inline blocks/b-big-carousel-quotes/b-big-carousel-quotes.css:1]: Pentru acest fisier nu trebuie sa avem si un scss?

Comment (Tiseanu) [reply blocks/b-big-carousel-quotes/b-big-carousel-quotes.css:1]: era nu stiu dece nu l-a adaugat, i-am dat acum inca un push

File: styles/styles.scss [hunk 1]
```diff
@@ -728,7 +735,7 @@ main .section.highlight {
     padding-top: 1em;
   }
 
-  main .default-content-wrapper h1, main .default-content-wrapper h2 {
+  main .default-content-wrapper h1, main .default-content-wrapper h2, main .default-content-wrapper h3 {
```
Comment (Matei-Iordache) [inline styles/styles.scss:738]: why is everything 23px ? What is the point of different tags if they have the same font size ?

Comment (Tiseanu) [reply styles/styles.scss:738]: pt ca zona de dafaul-content trebuie sa arate la fel in pagina, pt consecventa, chiar daca fol diferite tipuri de h, care vizeaza mai mult zona de seo

Comment (Tiseanu) [reply styles/styles.scss]: am actualizat

**Review by Matei-Iordache — APPROVED** [review]
LGTM

**Pull request merged**